app/views/fields/hours_field/_form.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="field-unit__label">
-  <label for="building_hours">Slug</label>
+  <label for="building_hours">Hours Identifier</label>
```

Comment:
@cdoyle-temple We should follow the [administrate practice](https://github.com/thoughtbot/administrate/blob/master/app/views/fields/has_many/_form.html.erb#L23) of dynamically defining this label otherwise, it will be `label for="building_hours"` even in the spaces form, which would be incorrect. 
```suggestion
  <%= f.label field.attribute, for: "#{f.object_name}_#{field.attribute_key}" %>
```
It does mean you will have to define a translation for these forms as well. But it will provide flexibility that they could be different across forms.